avocado/utils/softwareraid.py

```diff
@@ -165,3 +165,16 @@ def stop(self):
         """
         cmd = f"mdadm --manage {self.name} --stop"
         return self._run_command(cmd, log_details=False)
+
+    def exists(self):
+        """
+        checks if softwareraid exists or not
+
+        :return: True if exists, False otherwise.
+        :rtype: bool
+        """
+        cmd = f'mdadm --detail --test {self.name}'
+        result = process.run(cmd, shell=True, ignore_status=True)
```

Comment:
mdadm --detail:

mdadm: must be super-user to perform this action

I guess we need to make a note at the docstring saying that you need to be superuser to run this method, and also call process.run() with `sudo=True`.

Reply:
> mdadm --detail:
> 
> mdadm: must be super-user to perform this action
> 
> I guess we need to make a note at the docstring saying that you need to be superuser to run this method, and also call process.run() with `sudo=True`.

@beraldoleal Thanks for pointing this out.
Added the changes. Please re-review and merge if no more comments.